garage/tf/algos/off_policy_rl_algorithm.py

```diff
@@ -87,3 +82,61 @@ def get_itr_snapshot(self, itr, samples_data):
     def optimize_policy(self, itr, samples_data):
         """Optimize policy network."""
         raise NotImplementedError
+
```

Comment:
is this really identical for all off-policy algorithms, or just DDPG? is DQN the same? what about SAC?

maybe you just want to inherit from DDPG instead.

Reply:
That makes sense.